moto/sqs/models.py

```diff
@@ -327,7 +327,7 @@ def _set_attributes(self, attributes, now=None):
             if key in integer_fields:
                 value = int(value)
             if key in bool_fields:
-                value = value == "true"
+                value = value == "true" or value == "True"
```

Comment:
```suggestion
                value = value.lower() == "true"
```
This might be a better approach, to catch all possibilities - assuming value is always a string?